tests/test_custom_tokens.py

```diff
@@ -0,0 +1,97 @@
+import copy
+
+import pytest
+import torch
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from peft import AutoPeftModel, CustomTokensConfig, LoraConfig, get_peft_model
+
+
+class TestCustomTokens:
+    @pytest.fixture
+    def model_id(self):
+        return "trl-internal-testing/tiny-random-LlamaForCausalLM"
+
+    @pytest.fixture
+    def model(self, model_id):
+        return AutoModelForCausalLM.from_pretrained(model_id)
+
+    @pytest.fixture
+    def tokenizer(self, model_id):
+        return AutoTokenizer.from_pretrained(model_id)
+
+    def test_stand_alone_usage(self, model, tokenizer, tmp_path):
+        original_model = copy.deepcopy(model)
+        peft_config = CustomTokensConfig(target_modules=["embed_tokens"], token_indices=[0, 1, 2])
```

Comment:
Just wondering: Would the test cover corner cases a little better if `token_indices` were not consecutive tokens starting at 0? So e.g. `[1, 3]` instead?